telos-frontend/src/components/calendar/DashboardCalendar.jsx

```diff
@@ -1,36 +1,159 @@
 import { Calendar, momentLocalizer } from 'react-big-calendar';
+import { useState } from 'react';
+import TextField from '@material-ui/core/TextField';
 import moment from 'moment';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import 'fontsource-roboto';
 import 'react-big-calendar/lib/css/react-big-calendar.css';
 import './react-big-calendar.css';
 
 const localizer = momentLocalizer(moment);
 
-const events = [
-  {
-    id: 14,
-    title: 'Today',
-    start: new Date(new Date().setHours(new Date().getHours() - 3)),
-    end: new Date(new Date().setHours(new Date().getHours() + 1)),
-  },
-  {
-    id: 23,
-    title: 'Go to the gym 💪',
-    start: new Date(new Date().setHours(new Date().getHours() - 3)),
-    end: new Date(new Date().setHours(new Date().getHours() + 5)),
-  },
-];
+const sendAddEventRequest = () => {
+  const a = 'temp';
```

Comment:
Please use a nice name :)

Reply:
Yeah, that is just a test function which I'll code up once I integrate the backend into it. This will make the API requests to add data.

Reply:
Can you add a comment to talk about that then?